frontend/src/components/Town/interactables/Music/StepSequencer.tsx

```diff
@@ -168,8 +248,142 @@ export default function MusicArea({
 
   /*_______________________________________________________________________*/
 
+  /* Whenever the user presses a button, pause the game and clear the error log (error) if it exists */
+  const handleKeyDown = (event: { key: string }) => {
+    if (error !== '') {
+      setError('');
+    }
+    if (event.key === 'Enter') {
+      console.log('TODO: Search the database for the title!');
+    } else {
+      coveyTownController.pause();
+    }
+  };
+
+  /* Whenever the user releases a button, unpause the game*/
+  const handleKeyUp = () => {
+    // TODO: This is broken, there's some logic here that's wrong (compare it to the basement dining table below and the code at /components/Town/interactables/NewConversationModal.tsx)
```

Comment:
Nit: multiple Todo's are included throughout this pr, is it possible to list those at the top of each file in order to keep track of which functionalities are not fully implemented yet. 
EX. Saving the song information after the save checks

Reply:
Done!